### PR TITLE
Remove checks for existing translator

### PR DESCRIPTION
**Pull request by greg0ire**
I am targetting this branch, because I think this is BC

## Changelog

```markdown
### Removed
- The admin no longer checks for the `translator` service before translating.
```
## Subject

To use SonataAdminBundle, the translator service has to be enabled. It
is always injected in the admin by a compiler pass, which means there is
no point (anymore?) in checking it exists.

Closes #4089

### CODE REVIEW

File: Tests/Controller/CRUDControllerTest.php [hunk 1]
```diff
@@ -315,7 +315,7 @@ protected function setUp()
                     return true;
                 }
 
-                if ($id == 'security.csrf.token_manager' && Kernel::MAJOR_VERSION >= 3  && $tthis->getCsrfProvider() !== null) {
+                if ($id == 'security.csrf.token_manager' && Kernel::MAJOR_VERSION >= 3 && $tthis->getCsrfProvider() !== null) {
```
Comment (OskarStark) [inline Tests/Controller/CRUDControllerTest.php]:
```diff
- if ($id == 'security.csrf.token_manager' && Kernel::MAJOR_VERSION >= 3 && $tthis->getCsrfProvider() !== null) {
+ if ('security.csrf.token_manager' == $id && Kernel::MAJOR_VERSION >= 3 && null !== $tthis->getCsrfProvider()) {
```

and pay attention!!! `$tthis` looks wrong!

Comment (greg0ire) [inline Tests/Controller/CRUDControllerTest.php]: The `$this` part is intentional, and the Yoda is not a StyleCI fix

Comment (OskarStark) [inline Tests/Controller/CRUDControllerTest.php]: you are right, didn't get this with the `$tthis` 👍

**Comment by OskarStark** [comment]
minor or patch? @greg0ire

**Comment by greg0ire** [comment]
I'd say patch because we're not touching the way the software can be used : it already could not be used without translation.

**Comment by OskarStark** [comment]
shall i squash merge it @greg0ire ?

**Comment by greg0ire** [comment]
> shall i squash merge it @greg0ire ?

No, please merge it the regular way.

**Comment by OskarStark** [comment]
Thanks @greg0ire 👍

**Pull request merged**